.github/workflows/release.yml

```diff
@@ -2,7 +2,7 @@ name: Release PKE
 on:
   push:
     tags:
-      - "[0-9]+.[0-9]+.[0-9]+"
+      - "*"
```

Comment:
Note: I would rather support a [restricted pattern](https://github.com/banzaicloud/cloudinfo/blob/master/.github/workflows/docker.yaml#L9).

Reply:
I should have specified in the PR description that I do not intend to merge this, just release a dev tag with the help of it...

Reply:
I see dev tags as a useful feature for the future, that's why I proposed having a pattern similar to cloudinfo for example.